src/components/hooks.ts

```diff
@@ -0,0 +1,17 @@
+import { useEffect, RefObject } from 'react';
```

Comment:
Is this used anywhere? I grep'd for it and i can't find any uses. Delete file/function if not used.